piton/design/chipset/xilinx/xupp3r/devices.xml

```diff
@@ -0,0 +1,63 @@
+<!--
```

Comment:
I'm not sure that XUPP3R is going to act as a chipset board. We can probably leave out these files in chipset/xilinx/xupp3r for now